.buildkite/scripts/bintray_upload.sh

```diff
@@ -40,12 +49,15 @@ hab_artifact=$(buildkite-agent meta-data get "hab-artifact")
 #
 # -s = skip publishing
 # -r = the repository to upload to
-sudo -E HAB_BLDR_CHANNEL="${channel}" \
-                hab pkg exec core/hab-bintray-publish \
-                publish-hab \
-                -s \
-                -r stable \
-                "/hab/cache/artifacts/${hab_artifact}"
+sudo HAB_BLDR_CHANNEL="${channel}" \
```

Comment:
next time, it might not hurt to try this with an `env` to formally handle the environment variables like: `sudo env HAB_BLDR_CHANNEL=……`.